index.html

```diff
@@ -0,0 +1,175 @@
+              <span>Loyalty Program</span>
+            </a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="office-header">
+      <div class="container">
+        <div class="row mid-header">
+          <div class="header-logo col-12 col-md-3">
+            <a href="#" aria-label="main page link" class="logo_link"><img src="./images/header_logo.png"
+                alt="logo"></a>
+          </div>
+          <div class="col-12 col-md-4 offset-md-1">
+            <form class="search-form">
+              <input class="search-form__text" type="search" name="search" placeholder="Search Location">
+              <button class="search-form__btn" type="submit" value="search">
+                <span class="icon icon-search search-form__btn--icon"></span>
+              </button>
+            </form>
+          </div>
+          <div class="col-1 offset-md-1">
+            <a href="#" aria-label="go to shopping list"><span class="icon icon-shopping-cart shop-cart"></span>
+            </a>
+          </div>
+          <div class="col-2">
+            <a class="dropdown"><img src="./images/avatar.png" alt="user photo">
+              <span class="dropdown__user-name">Nathan</span>
+              <span class="icon icon-Stroke dropdown__stroke "></span>
+            </a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <nav class="nav">
+      <div class="container">
+        <div class="row nav-header">
+          <div class="bot-header">
+            <ul class="nav-list">
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#">All Deals</a>
+              </li>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#">Hotels</a>
+              </li>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#">Activities</a>
+              </li>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#">Hotel Day Packages</a>
+              </li>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#">Restaurants</a>
+              </li>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#">Events</a>
+              </li>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#">Rodrigues</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </nav>
+    </div>
+    </div>
+  </header>
+
+  <main>
+    <section class="section top-section">
+      <div class="container">
+        <nav aria-label="breadcrumb">
+          <ol class="breadcrumb">
+            <li class="breadcrumb-item"><a href="#">Home</a></li>
+            <li class="breadcrumb-item"><a href="#">Flash Offers</a></li>
+            <li class="breadcrumb-item" aria-current="page">Grand Hilton Hotel</li>
+          </ol>
+        </nav>
```

Comment:
We should use `nav` tag only for main navigation.

Reply:
> We should use `nav` tag only for main navigation.

corrected